wiz-outpost-lite/values.yaml

```diff
@@ -77,3 +79,4 @@ runners:
     enabled: false
     image:
       name: outpost-lite-runner-vcs
+    terminationGracePeriodSeconds: 300 # 5 minutes
```

Comment:
Not used?

Reply:
Each runner's values automatically override the global ones, I've introduced some hierarchy in the _helpers.tpl file that does this, so this overrides the global specifically for PR scans